Documentation/data-collection.md

```diff
@@ -149,10 +149,10 @@ data:
     # series associated with the csv will be deleted.
     - '{__name__="csv_abnormal"}'
     # (@openshift/ocs-operator, OCS-storage-management-team) Generic Storage metrics:
-    # cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum gives 
+    # cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum gives
     # the total amount of storage requested by PVCs from a particular storage provisioner in bytes.
     - '{__name__="cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum"}'
-    # cluster:kubelet_volume_stats_used_bytes:provisioner:sum will gives 
+    # cluster:kubelet_volume_stats_used_bytes:provisioner:sum will gives
```

Comment:
funky, the `generate` target wants to introduce the trailing whitespace. do you mind to add it back in ?

Reply:
@s-urbaniak I have now added the space back in. I did this by running `make docs` on my local clone, as per @paulfantom's guidance when this happened previously. That seems to refresh the embedded reference to `../manifests/0000_50_cluster_monitoring_operator_04-config.yaml `within `Documentation/data-collection.md`.